packages/vanilla/src/components/select/select.stories.js

```diff
@@ -0,0 +1,106 @@
+export default {
+  title: 'Patterns',
+  parameters: {
+    html: {
+      root: '.cbp-form'
+    }
+  },
+  argTypes: {
+    label: {
+      name: 'Label Element',
+      description: 'Represents a caption for the `<select>` element in the user interface.',
+      control: { type: 'text' }
+    },
+    inputDescription: {
+      name: 'Input Description',
+      description: 'Instructions or supplementary information regarding the `<select>` element.',
+      control: { type: 'text' }
+    },
+    labelFor: {
+      name: 'Label `for` Attribute',
+      description: 'Indicates the form element that this `<label>` describes and has the value which is the `id` of the `<select>` element it relates to.',
+      control: { type: 'text' }
+    },
+    formControlName: {
+      name: 'Input `name` Attribute',
+      description: 'Name of the form control in the input element. Submitted with the form as part of a name/value pair.',
+      control: { type: 'text' }
+    },
+    placeholderOption: {
+      name: 'Placeholder Option Text',
+      description: 'Text of the placeholder option.',
+      control: { type: 'text' }
+    },
+    errorMessage: {
+      name: 'Error Message',
+      description: 'A message in the input description that a problem has occurred.',
+      control: { type: 'text' }
+    },
+    optionsObj: {
+      name: 'Options Object',
+      description: 'Sets the label of the `<option>` and the `value` attribute.',
+      control: { type: 'object' }
+    },
+    disabled: {
+      name: '`disabled` Attribute',
+      description: 'Attribute that indicates that the user cannot interact with the control.',
+      control: { type: 'boolean' }
+    }
+  },
+  decorators: [
+    (Story, context) => `
+      <form class="cbp-form">
+        <label for=${context.args.labelFor} class="cbp-input__label">${context.args.label}</label>
+        <p class="cbp-input__description" ${!context.args.required ? '' : 'hidden'}>${context.args.inputDescription}</p>
+        <p class="cbp-input__description ${context.args.required ? 'cbp-input__description--error': ''}" ${context.args.required ? '' : 'hidden'}><i class="fas fa-exclamation-triangle"></i>${context.args.errorMessage}</p>
+        ${Story().outerHTML || Story()}
+      </form>
+    `
+  ]
+}
+
+const Template = ({ labelFor, formControlName, placeholderOption, disabled, optionsObj: { option1, option2, option3, option4, option5} }) => (
+  `
+    <select class="cbp-input__select" name=${formControlName} id=${labelFor} ${disabled ? 'disabled' : ''}>
+      <option value="" selected>${placeholderOption}</option>
```

Comment:
We don't need `selected` on the first option. It will be the default anyway.

The presentation of this option looks like something is selected. I think we need to establish a best practice here rather than make it an editable control/option. At SSA we used "--" for the default/no selection because it cannot be mistaken for a selection. USWDS uses "- Select -" which is also common. Because the label says what the thing is being selected, we don't need to repeat it here though - either of the above options would be a better choice and we should not let the user customize it in Storybook to make bad decisions.

Reply:
Removed the control and replaced with `-- Select --`